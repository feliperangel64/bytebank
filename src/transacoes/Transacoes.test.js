import React from "react";
import { render } from "@testing-library/react";
import Transacao from "./Transacao";

describe("Componente de Transações", () => {
  it("O snapshot do componente deve permanecer sempre o mesmo", () => {
    const { container } = render(
      <Transacao data="15/04/2021" tipo="saque" valor="20.00" />
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
