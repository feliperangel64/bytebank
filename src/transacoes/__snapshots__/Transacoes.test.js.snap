// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Componente de Transações O snapshot do componente deve permanecer sempre o mesmo 1`] = `
<div
  class="transacao-container"
>
  <p>
    15/04/2021
  </p>
  <p>
    saque
  </p>
  <p>
    R$ 
    20.00
  </p>
</div>
`;
